### PR TITLE
v1.8: Search cutoff

### LINKED ISSUE #2798: v1.8: Search cut-off
v1.8 sets a 1500ms timeout to prevent search requests from taking too long. This value can be customized with a new index setting sub-route.

## Tasks
- [ ] Update index settings page
- [ ] Create new index subroutes page

## Reference
- PRD: https://meilisearch.notion.site/Search-cutoff-usage-fb5e9a07a7624965a541d0a6f8eb37bc
- engine issue: https://github.com/meilisearch/meilisearch/issues/4488
- SME: @irevoire

### PR DESCRIPTION
**Pull request by guimachiavelli**
Closes #2798

### CODE REVIEW

**Review by curquiza — APPROVED** [review]

**Review by ManyTheFish — APPROVED** [review]

**Review by ManyTheFish — APPROVED** [review]

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -1187,3 +1188,14 @@ index_settings_tutorial_api_put_setting_1: |-
 index_settings_tutorial_api_task_1: |-
   curl \
     -X GET 'http://localhost:7700/tasks/TASK_UID'
+get_search_cut_off_1: |-
```
Comment (curquiza) [inline .code-samples.meilisearch.yaml:1191]: I was thinking, maybe `get_search_cutoff_1` is more consitent with the naming?

Comment (guimachiavelli) [reply .code-samples.meilisearch.yaml:1191]: hmmmm…you lead me down a spelling rabbit hole. 

I was using `cut-off`, as suggested by the Oxford English Dictionary (aka OED). According to that spelling, `get_search_cut_off` would be more accurate.

However OED is heavily skewed towards British English. The Merriam-Webster and the New Oxford dictionaries both lean American and prefer `cutoff`. Since our documentation is written in American English, I suppose it's better to follow that spelling, even if both are valid.

So, in short, yes, you are correct.

Comment (curquiza) [reply .code-samples.meilisearch.yaml:1191]: I was saying "consistent" based on OUR naming 😊 which is `searchCutoffMs` and not `SearchCutOffMs`

Comment (guimachiavelli) [reply .code-samples.meilisearch.yaml]: That's also true. And, I guess, another reason to use the preferred American spelling.
